src/tic_tac_toe/display.clj

```diff
@@ -0,0 +1,20 @@
+(ns tic-tac-toe.display)
+
+(defn stringify-board
+  [board]
```

Comment:
Can you keep the formatting consistent? Will the parameters be after the function name or will it be in the line below? Board and core appears to have it on the same line but you did a switcharoo here.